resource/process.go

```diff
@@ -19,17 +19,21 @@ func (p *Process) SetID(id string) { p.Executable = id }
 func (p *Process) GetTitle() string { return p.Title }
 func (p *Process) GetMeta() meta    { return p.Meta }
 
-func (p *Process) Validate(sys *system.System) []TestResult {
+func (p *Process) Validate(sys *system.System) ([]TestResult, error) {
+	var results []TestResult
 	skip := false
-	sysProcess := sys.NewProcess(p.Executable, sys, util.Config{})
+	sysProcess, err := sys.NewProcess(p.Executable, sys, util.Config{})
+	if err != nil {
+		return results, err
```

Comment:
Here I have to check the error, resource.Process does not let Validate() return any error, whats the right way to construct a Validatevalue here for this situation where the sys.NewProcess fails?

Reply:
This can fail because sys.GetProcs can fail in the go-ps package

Reply:
OK I managed to resolve this issue while keeping the error handling, so good for review with all tests green `The command "make all" exited with 0.`